lucene/analysis/common/src/test/org/apache/lucene/analysis/hunspell/TestHunspellRepositoryTestCases.java

```diff
@@ -64,7 +74,12 @@ public TestHunspellRepositoryTestCases(String testName, Path pathPrefix) {
   }
 
   @Test
-  public void test() throws IOException, ParseException {
-    SpellCheckerTest.checkSpellCheckerExpectations(pathPrefix, false);
+  public void test() throws Throwable {
+    ThrowingRunnable test = () -> SpellCheckerTest.checkSpellCheckerExpectations(pathPrefix);
+    if (IGNORED.contains(testName)) {
```

Comment:
It should perhaps read FAILING instead of ignored if you're still running the test and testing the outcome?

Reply:
Renamed, thanks!